tests/pytests/test_mcmurchie_davidson.py

```diff
@@ -0,0 +1,38 @@
+"""
+This file tests one-electron integrals from libmints computed with
+the McMurchie-Davidson scheme
+"""
+import numpy as np
+import pytest
+import psi4
+from pathlib import Path
+import json
+
+
+pytestmark = pytest.mark.quick
```

Comment:
```suggestion
pytestmark = [pytest.mark.psi, pytest.mark.api, pytest.mark.quick]
```
prep for a future PR